docs/RepoStructure.md

```diff
@@ -0,0 +1,72 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Repositories
+
+This respository contains the following:
```

Comment:
Worth mentioning that there are build related things (the release scripts, the calico/build image, the calico/test image)